core/modules/mysql/MySqlConfig.h

```diff
@@ -60,16 +60,14 @@ class MySqlConfig {
      *  @param port:         MySQL port
      *  @param socket:       MySQL socket
      *  @param db:           MySQL database
-     *  @param maxtablesize: maximum table size.
      *
      * @throws std::runtime_error: if checkValid is true and some parameters are invalid
      */
     MySqlConfig(std::string const& username, std::string const& password,
                 std::string const& hostname,
                 unsigned int const port,
                 std::string const& socket,
-                std::string const& db = "",
-                size_t maxresultsize = 0);
+                std::string const& db = "");
```

Comment:
I think the C+ Style guide allows no spaces around the assignment operator here.